proposals/0344-distributed-actor-runtime.md

```diff
@@ -1074,37 +1071,37 @@ final struct ClusterSystem: DistributedActorSystem {
     // Type substitutions matter only for distributed methods which use generics:
     var genericSubstitutions: [String]
 
-    // For illustration purposes and simplicity of code snippets we use '[Data]' here,
-    // but real implementations can be much more efficient here -- packing all the data into exact
-    // byte buffer that will be passed to the networking layer etc.
+    // For illustration purposes and simplicity of code snippets we use `[Data]` here,
+    // but real implementations can be much more efficient here — packing all the data into exact
+    // byte buffer that will be passed to the networking layer, etc.
     var arguments: [Data] // example is using Data, because that's what Codable coders use
 
     // Metadata can be used by swift-distributed-tracing, or other instrumentations to carry extra information:
-    var metadata: [String: [Data]] // additional metadata, such as trace-ids etc.
+    var metadata: [String: [Data]] // additional metadata, such as trace-ids
   }
 }
 ```
 
 Note that `method` property is enough to identify the target of the call, we do not need to carry any extra type information explicitly in the call. The method identifier is sufficient to resolve the target method on the recipient, however in order to support generic distributed methods, we need to carry additional (mangled) type information for any of the generic parameters of this specific method invocation. Thankfully, these are readily provided to us by the Swift runtime, so we'll only need to store and send them over.
 
 > **Note:** An implementation may choose to define any shape of "envelope" (or none at all) that suits its needs. It may choose to transport mangled names of involved types for validation purposes, or choose to not transfer them at all and impose other limitations on the system and its users for the sake of efficiency.
->
-> While advanced implementations may apply compression and other techniques to minimize the overhead of these envelopes - this is a deep topic by itself, and we won't be going in depth on it in this proposal - rest assured though, we have focused on making different kinds of implementations possible with this approach.
+> 
```

Comment:
Is this considered a trailing whitespace as well? There are 3 more occurrences of this later on.

Reply:
I double-checked CommonMark spec and believe this is a valid (and somehow recommended) use case. So I tend to remove the trailing whitespace here.

Reply:
Alright then, sounds good!